homework/index.js

```diff
@@ -0,0 +1,64 @@
+function getPromise() {
+  const ret = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+    .then(response => response.json());
+  return ret;
+}
+
+
+function populate(Prom) {
+  const selectContent = document.getElementById('liste').value;
+  Prom.then((result) => {
+    result.forEach((x) => {
+      if (x.name === selectContent) {
+        document.getElementById('show').innerHTML = `
```

Comment:
👏 for string interpolation & starting on a new line! However, I wouldn't use more than one space anywhere. 
(And nit-pick: in English, we don't type a space *before* colons ;) )

Reply:
Another remark, more importantly: instead of doing the combination of `forEach()` and then checking the name; you can use `filter()` to find the appropriate repo and work with it! ;)

Reply:
And another detail: instead of using `x` as a name, call it what it is: a `repo`!